### PR TITLE
Heretic slider values + high mouse sensitivity support

### PR DESCRIPTION
**Pull request by Zodomaniac**
Done the fashion it was in Doom, had to introduce an extra variable though to make it work.

### CODE REVIEW

File: src/heretic/mn_menu.c [hunk 1]
```diff
@@ -1838,8 +1844,13 @@ static void DrawSlider(Menu_t * menu, int item, int width, int slot)
                                            : "M_SLDMD2"), PU_CACHE));
     }
     V_DrawPatch(x2, y, W_CacheLumpName(DEH_String("M_SLDRT"), PU_CACHE));
-    V_DrawPatch(x + 4 + slot * 8, y + 7,
+    V_DrawPatch(x + 4 + position * 8, y + 7,
                 W_CacheLumpName(DEH_String("M_SLDKB"), PU_CACHE));
+
+    // [crispy] print the value
+    M_snprintf(num, 4, "%3d", slot);
+    MN_DrTextA(num, x2 + 32, y + 3);
```
Comment (Zodomaniac) [inline src/heretic/mn_menu.c:1852]: Why `x2+32` here
HTIC00

File: src/heretic/mn_menu.c [hunk 2]
```diff
@@ -1844,13 +1840,19 @@ static void DrawSlider(Menu_t * menu, int item, int width, int slot)
...
     MN_DrTextA(num, x2 + 32, y + 3);
 
+    // [crispy] do not crash anymore if the value is out of bounds
+    if (slot >= width)
+    {
+        slot = width - 1;
+    }
+
+    V_DrawPatch(x + 4 + slot * 8, y + 7,
+                W_CacheLumpName(DEH_String("M_SLDKB"), PU_CACHE));
```
Comment (Zodomaniac) [inline src/heretic/mn_menu.c:1855]: Correcting my noobish mistakes, as usual

File: src/heretic/mn_menu.c [hunk 3]
```diff
@@ -1001,10 +1001,11 @@ static boolean SCMouseSensi(int option)
 {
     if (option == RIGHT_DIR)
     {
-        if (mouseSensitivity < 9)
```
Comment (fabiangreffrath) [inline src/heretic/mn_menu.c:1004]: Please keep a reasonable limit. 255 ought to be enough for everyone. :wink:

File: src/heretic/mn_menu.c [hunk 4]
```diff
@@ -1001,10 +1001,11 @@ static boolean SCMouseSensi(int option)
...
-        {
-            mouseSensitivity++;
-        }
+		//[crispy] remove mouse sensitivity limit
```
Comment (fabiangreffrath) [inline src/heretic/mn_menu.c:1004]: `// [`
You forgot a space.

File: src/heretic/mn_menu.c (hunk 3 above)
Comment (fabiangreffrath) [reply src/heretic/mn_menu.c:1004]: As in Doom:
https://github.com/fabiangreffrath/crispy-doom/blob/153d796a691cf2073e85ce997b0f7623a120492f/src/doom/m_menu.c#L1827

File: src/heretic/mn_menu.c [hunk 5]
```diff
@@ -1001,9 +1001,10 @@ static boolean SCMouseSensi(int option)
 {
     if (option == RIGHT_DIR)
     {
-        if (mouseSensitivity < 9)
+		// [crispy] remove mouse sensitivity limit
+        if (mouseSensitivity < 255)
         {
-            mouseSensitivity++;
+			mouseSensitivity++;
```
Comment (fabiangreffrath) [inline src/heretic/mn_menu.c:1007]: White space clutter. Please keep this clean.

File: src/heretic/mn_menu.c [hunk 6]
```diff
@@ -1001,9 +1001,10 @@ static boolean SCMouseSensi(int option)
 {
     if (option == RIGHT_DIR)
     {
-        if (mouseSensitivity < 9)
+		// [crispy] remove mouse sensitivity limit
```
Comment (fabiangreffrath) [inline src/heretic/mn_menu.c:1004]: Comment indentation.

**Pull request merged**